packages/bot/src/components/transactions.jsx

```diff
@@ -0,0 +1,169 @@
+// import className from 'classnames';
+import { Money,Popover } from 'deriv-components';
+import { PropTypes }     from 'prop-types';
+import React             from 'react';
+import { Scrollbars }    from 'tt-react-custom-scrollbars';
+import {
+    BuyPriceIcon,
+    ExitSpotIcon,
+    EntrySpotIcon,
+    PendingIcon,
+    RefrenceIdIcon,
+    SettledIcon,
+}                        from './Icons.jsx';
+import IconTradeType     from './icon-trade-types.jsx';
+import { connect }       from '../stores/connect';
+import { translate }     from '../utils/tools';
+import '../assets/sass/transactions.scss';
+
+const Transaction = ({ contract }) => {
+    return (
+        <table className='transactions__item'>
+            <tbody>
+                <tr className='transactions__tr'>
+                    <td className='transactions__middle transactions__td'>
+                        <Popover
+                            className='transactions__inline transactions__top'
+                            alignment='left'
+                            message={translate(contract.contract_type)}
```

Comment:
- We don't need to `translate` the contract type, this is done by the API itself.
- The `translate` will never be able to resolve this value since it's external.